lexicons/tools/ozone/moderation/defs.json

```diff
@@ -619,6 +630,58 @@
         "height": { "type": "integer" },
         "length": { "type": "integer" }
       }
+    },
+    "accountHosting": {
+      "type": "object",
+      "required": ["status"],
+      "properties": {
+        "status": {
+          "type": "string",
+          "knownValues": ["takendown", "suspended", "deleted", "deactivated"]
+        },
```

Comment:
I see the value `unknown` used in the implementation—may be worth adding here and to the same field in `#recordHosting`.